docs/generated/changelog.html

```diff
@@ -32,9 +32,11 @@ <h2>Version x.x.x</h2>
           in the type table
         </li>
         <li>
-          feat: support composite_query in candid
-          fix: fix a bug in decoding service types, when function types come after the service type in the type table
+          feat: support composite_query in candid fix: fix a bug in decoding service types, when
```

Comment:
Looks like the newline was accidentally removed here.

Reply:
This was done by the a pre-commit hook, might be related to having two types of changes under the same `li`, i'll one to another `li`